conf/log4j-dev.properties

```diff
@@ -8,27 +8,42 @@
 
 log4j.threshhold=OFF
 
-#log4j.rootLogger=WARN,A1
-log4j.logger.com.zimbra=WARN,A1
+# rootLogger.level=WARN, A1
 
```

Comment:
Plz also add appender for root logger and level too.
rootLogger.level = WARN
rootLogger.appenderRef.stdout.ref = ConsoleLogger